### DISCUSSION TITLE
Redis 란 무엇인가

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#13] [Q&A] [answered]
Redis 란 무엇인가

**Comment by tjdud22** [comment, marked as answer]
Key, Value 구조의 비정형 데이터를 저장하고 관리하기 위한 오픈 소스 기반의 비관계형 데이터 베이스 관리 시스템 (DBMS)입니다.

데이터베이스, 캐시, 메세지 브로커로 사용되며 인메모리 데이터 구조를 가진 저장소입니다.
